### ISSUE TITLE
Integrate the Evolutions API into the webapp

### ISSUE THREAD

**Issue by collinsnji** [anshumanv/evolutions-webapp#9]
##### This issue is about using the API in the webapp. See #4 

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by collinsnji** [comment]
Is someone working on this? I'll like to begin work on this 😺 @anshumanv @sneakycr0w ??

**Comment by anshumanv** [comment]
@collinsnji check gitter please, there's a problem.
Also get started with this ! 👍 😄

**Comment by collinsnji** [comment]
Just checked Gitter. Seems like @joshuahunt10 is already working on this :). I'll let him push what he has before I continue :) @anshumanv can you add @joshuahunt10 as contributor and assign this issue to him instead? @joshuahunt10 are you OK with this?

**Comment by anshumanv** [comment]
A basic prototype is working at - http://evolutions.surge.sh/
